pubspec.yaml

```diff
@@ -14,10 +14,10 @@ dependencies:
   dio: ^4.0.4
   sqflite: ^2.0.1
   provider: ^6.0.1
-  intl: ^0.17.0
-  analyzer: ^4.1.0
-  collection: ^1.15.0
-  share_plus: ^4.0.4
+  intl: ^0.18.0
```

Comment:
Hey @dwikurniantom, `intl: ^0.18.0` is still not available in the stable channel, can you revert it back to `^0.17.0`?